app/src/components/inputs/inputConditionalAttributes/inputConditionalAttributes.jsx

```diff
@@ -0,0 +1,165 @@
+/*
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import { injectIntl } from 'react-intl';
+import {
+  CONDITION_HAS,
+  CONDITION_NOT_HAS,
+  CONDITION_ANY,
+  CONDITION_NOT_ANY,
+} from 'components/filterEntities/constants';
+import { getInputConditions } from 'common/constants/inputConditions';
+import { AttributeEditor } from 'components/main/attributeList/editableAttribute/attributeEditor';
+import { AttributeListField } from 'components/main/attributeList';
+import styles from './inputConditionalAttributes.scss';
+
+const cx = classNames.bind(styles);
+
+@injectIntl
+export class InputConditionalAttributes extends Component {
+  static propTypes = {
+    intl: PropTypes.object.isRequired,
+    value: PropTypes.object,
+    conditions: PropTypes.arrayOf(PropTypes.string),
+    onChange: PropTypes.func,
+    onFocus: PropTypes.func,
+    onBlur: PropTypes.func,
+    valueURLCreator: PropTypes.func,
+    keyURLCreator: PropTypes.func,
+    projectId: PropTypes.string,
+  };
+
+  static defaultProps = {
+    value: {},
+    onChange: () => {},
+    onFocus: () => {},
+    onBlur: () => {},
+    valueURLCreator: () => {},
+    keyURLCreator: () => {},
+    conditions: [CONDITION_HAS, CONDITION_NOT_HAS, CONDITION_ANY, CONDITION_NOT_ANY],
+    projectId: '',
+  };
+  state = {
+    opened: false,
+    attributes: this.props.value.attributes,
+  };
+
+  componentDidMount() {
+    document.addEventListener('click', this.handleClickOutside);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('click', this.handleClickOutside);
+  }
+
+  onClickConditionBlock = () => {
+    this.setState({ opened: !this.state.opened });
+  };
+
+  onClickConditionItem = (condition) => {
+    if (condition.value !== this.props.value.condition) {
+      this.setState({ opened: false });
+      this.props.onChange({
+        value: this.props.value.value,
+        condition: condition.value,
+      });
+    }
+  };
+
+  onChangeTags = (tags) => {
+    this.state.attributes.push({ key: tags.key, value: tags.value });
```

Comment:
Do not mutate state directly (https://reactjs.org/docs/state-and-lifecycle.html#using-state-correctly)